### ISSUE TITLE
Skript und Anleitung aktuell?

### ISSUE THREAD

**Issue by Siggi0904** [Pittini/iobroker-nodemihome#73]
Hallo Pittini,
Ist das Skript, die Anleitung und node-mihome noch aktuell?

Bei mir liegen die js-files mit dem javascript-Adapter 8.7.6 unter
`/opt/iobroker/node_modules/@iobroker-javascript.0/node-mihome/lib/devices`

Des Weiteren habe ich das Problem, dass bei jedem Update eines Plugins
oder vom javascript selbst meine eigenen Definitionsdateien wieder gelöscht sind.

Danke für ein Feedback.

**Comment by Pittini** [comment]
Ja, das mit dem JS Adapter 8.7.6 sich der Pfad geändert hat, ist mir zu Ohren gekommen, ich werd das hier mal anpinnen, selber hab ich noch net umgestellt, mein System is im freeze Status da alles läuft.
Das die eigenen Def DAteinen bei JS update nei einkopiert werden müssen, ist schon immer so und steht auch in der Anleitung. Dass das bei jedem Plugin update so ist, wäre mir neu, kann aber natürlich durch den neuen JS 8.7.6 so sein. Die 8.7.6 macht mit allerlei Skripten Probleme, ich will da mal abwarten ob da noch was gefixt wird.

**Comment by Siggi0904** [comment]
Hi Pittini,
ja, der JS macht scheinbar ein komplettes rebuild, wenn man Adapter aktualisiert oder installiert.
Daher sind die Benutzerdateien, also die eigenen Def Dateien dann immer weg.
Oder muss man die als Benutzer iobroker in den Ordner einfügen?

Danke, vielleicht findest du ja eine Lösung.
